io/src/test/scala/sbt/internal/io/DefaultWatchServiceSpec.scala

```diff
@@ -8,14 +8,12 @@ import scala.concurrent.duration._
 import scala.util.Properties
 
 object DefaultWatchServiceSpec {
-  // java.nio's default watch service is much slower on MacOS at the moment.
-  // We give it more time to detect changes.
-  val (pollDelay, maxWaitTime) = (50.milliseconds, 3.seconds)
+  val pollDelay =
+    Option(System.getProperty("io.watch.poll.ms")).fold(50.milliseconds)(_.toInt.milliseconds)
```

Comment:
could you prefix the prop key with `sbt.` please?

Reply:
I was only using this for travis debugging. It's gone now. I will keep in mind prefixing keys with sbt. if it comes up again though.